ci/before_deploy.sh

```diff
@@ -54,9 +54,17 @@ elif [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$ARCH" == "i386" ]; then
     # Make sure all files can be uploaded without permission errors
     sudo chown -R $USER:$USER "./target"
 elif [ "$TRAVIS_OS_NAME" == "windows" ]; then
-    choco install 7zip
+    choco install 7zip nuget.commandline
+    nuget install WiX
+
+    # Create zip archive
     7z a -tzip "./target/deploy/${name}-windows.zip" "./target/release/alacritty.exe" \
         "./target/release/winpty-agent.exe"
```

Comment:
I feel like it might be nice to rename this from `Alacritty-vX.X.X-windows.zip` to `Alacritty-vX.X.X-windows-portable.zip` (or `...-portable-windows.zip`). It seems like that would make the difference to the wix installer pretty clear and make it easy to figure out what the difference is.

Reply:
I think the .msi vs .zip extension should make this clear and that would be consistent with the way the rest of the releases are named. Also, "portable" usually means that the program is self-contained and writes config files into a relative directory.

Reply:
That's a great point, though the way the configuration file is generated is likely to change soon (it's only going to be generated on demand). So I feel like this would still help with telling the user what the difference is without having to download it.

I don't think there's any precedence with the current releases, since 'portable' installs aren't really something commonly used outside of Windows in my experience.

Reply:
Fixed.